Umbraco-Cloud/Upgrades/Manual-Upgrades/Manual-CMS-upgrade.md

```diff
@@ -14,117 +14,29 @@ Make sure you can run your Cloud project locally and restore content and media.
-    <add key="Umbraco.Deploy.ApiKey" value="9BEA9EAA7333131EB93B6DB7EF5D79709985F3FB" />
-
-**< connectionString >**
-
-    <connectionStrings>
-        <remove name="umbracoDbDSN" />
-        <add name="umbracoDbDSN" connectionString="Data Source=|DataDirectory|\Umbraco.sdf;Flush Interval=1;" providerName="System.Data.SqlServerCe.4.0" />
-        <!-- Important: If you're upgrading Umbraco, do not clear the connection string / provider name during your web.config merge. -->
-    </connectionStrings>
-
-**< umbraco.deploy >**
-
-    <umbraco.deploy>
-        <environments configSource="config\UmbracoDeploy.config" />
-        <settings configSource="config\UmbracoDeploy.Settings.config" />
-    </umbraco.deploy>
-
-:::note
-The following section with the `Dashboard.config` only applies to Umbraco 7 projects.
-:::
-
-### Dashboard.config
-
-When merging the `Dashboard.config` file make sure that you **do not overwrite / remove** the following settings:
-
-**Deploy**
-
-    <section alias="Deploy">
-        <areas>
-        <area>content</area>
-        </areas>
-        <tab caption="Your workspace">
-        <control>/App_Plugins/Deploy/views/dashboards/dashboard.html</control>
-        </tab>
-    </section>
-
-**StartupFormsDashboardSection**
-
-    <section alias="StartupFormsDashboardSection">
-        <areas>
-        <area>forms</area>
-        </areas>
-        <tab caption="Dashboard">
-        <control>/App_Plugins/umbracoforms/backoffice/dashboards/licensing.html</control>
-        <control>/App_Plugins/umbracoforms/backoffice/dashboards/yourforms.html</control>
-        <control>/App_Plugins/umbracoforms/backoffice/dashboards/activity.html</control>
-        </tab>
-    </section>
-
-Note that you **should not merge in** the following section from the new version of Umbraco:
-
-    <section alias="StartupDashboardSection">
-        <access>
-        <deny>translator</deny>
-        </access>
-        <areas>
-        <area>content</area>
-        </areas>
-        <tab caption="Get Started">
-        <access>
-            <grant>admin</grant>
-        </access>
-
-        <control showOnce="true" addPanel="true" panelCaption="">
-            views/dashboard/default/startupdashboardintro.html
-        </control>
-
-        </tab>
-    </section>
-
-### Other config files
-
-The following config files contain differences, and in most cases, you need to keep the ones from your Cloud project:
-
-* `/Splashes/noNodes.aspx`
-* `trees.config`
-* `umbracoSettings.config`
+```xml
+<ItemGroup>
+  <PackageReference Include="Umbraco.Cms" Version="9.0.1" />
+</ItemGroup>
+```
 
 ## Run the upgrade locally
 
-When you are done merging the new version of Umbraco with your clone of the Umbraco Cloud project, follow these steps to complete the upgrade:
+When you are upgrading the new version of Umbraco with your clone of the Umbraco Cloud project, follow these steps to complete the upgrade:
```

Comment:
Here I would suggest that you write:
"When you are done updating the nuget packages as mentioned above, follow these steps to complete the upgrade and verify that everything is working as expected before you push the changes to your Umbraco Cloud project"